### PR TITLE
feat: adding hamburger on sidebar

### PR DESCRIPTION
**Pull request by daptheHuman**
Reference on issue #1

### CODE REVIEW

File: src/components/Navbar/index.js [hunk 1]
```diff
@@ -28,6 +37,9 @@ export const Navbar = () => {
           <Link to='/theming'>Theming</Link>
         </li>
       </ul>
+      <div onClick={toggleNavbar} className='navbar-toggle'>
```
Comment (jpalacio0612) [inline src/components/Navbar/index.js:40]: It is considered bad practice to add an onClick event to an element other than a button. This is for semantic html and accessibility issues, maybe you could try something like:

```html
<button onClick={toggleNavbar} className='navbar-toggle'>
    <GiHamburgerMenu size={25} />
</button>
```

```css
.navbar-toggle {
  color: aliceblue;
  background: transparent;
  border: none;
  cursor: pointer;
  padding: 1em;
  top: 0;
  left: 0;
  position: absolute;
  display: fixed;
  transition: 0.5s;
  &:hover {
    background: #04001a;
  }
}
```

File: src/components/Navbar/styles.scss [hunk 2]
```diff
@@ -7,11 +7,27 @@
...
-  margin: 0 10px 0 10px;
-  padding: 20px;
+  flex-direction: column;
   color: white;
+  position: absolute;
+  top: 5%;
+  left: 0;
+  padding: 1rem 0 0 2rem;
+  display: flex;
+  width: 30vw;
```
Comment (jpalacio0612) [inline src/components/Navbar/styles.scss:26]: The issue with this measurement depending on the width of the screen is that mobile will look very tight. Maybe you could try two different width measures depending on a media query.

image

**Review by jpalacio0612 — CHANGES_REQUESTED** [review]
Thank you very much, I leave you a couple of comments about things that can be improved.

File: src/components/Navbar/styles.scss (hunk 2 above)
Comment (daptheHuman) [reply src/components/Navbar/styles.scss:26]: ok let me try again

**Comment by daptheHuman** [comment]
sorry I accidentally make 3 commits😔

**Review by jpalacio0612 — APPROVED** [review]
Looks good to me!
